src/argilla/client/feedback/unification.py

```diff
@@ -18,12 +18,14 @@
 from enum import Enum
 from typing import Any, Dict, List, Union
 
+import pandas as pd
```

Comment:
Is `pandas` included as a core Python package dependency? Or does it need to be installed separately?

Reply:
It already is included.